src/cube/box-number.tsx

```diff
@@ -0,0 +1,12 @@
+import { Avatar } from "@mui/material";
+import React from "react";
+
+interface BoxNumberProps {
+    number: number;
```

Comment:
The box number is actually a string. Just make this `ReactNode` for maximum flexibility.